src/components/Selector/Selector.js

```diff
@@ -146,43 +141,43 @@ const Selector = ({
     return null;
   };
 
-  const renderOption = (option: ?Option, disabled: boolean) => {
+  const renderOption = (option: ?Option) => {
     if (!option) return null;
     const {
-      name,
       imageUrl,
       lastUpdateTime,
-      imageSource,
     } = option;
+    let { name } = option;
 
-    const onItemPress = (itemImagePress: boolean = false) => {
-      if (itemImagePress && onOptionImagePress) {
-        onOptionImagePress(option);
-        return;
-      }
-      if (onOptionSelect) onOptionSelect(option);
-      setIsOptionsVisible(false);
-    };
+    if (isValidAddress(name)) {
+      name = t('ellipsedMiddleString', {
+        stringStart: name.slice(0, 6),
+        stringEnd: name.slice(-6),
+      });
+    }
+
+    const updatedUserImageUrl = lastUpdateTime && imageUrl ? `${imageUrl}?t=${lastUpdateTime}` : imageUrl;
 
     return (
-      <ListItemWithImage
-        label={name}
-        disabled={disabled}
-        onPress={() => !disabled && setIsOptionsVisible(true)}
-        avatarUrl={imageUrl}
-        navigateToProfile={() => onItemPress(true)}
-        imageUpdateTimeStamp={lastUpdateTime}
-        itemImageSource={imageSource}
-        padding="0 14px"
-      />
+      <SelectedOption>
+        <ProfileImage
+          uri={updatedUserImageUrl}
+          userName={name}
+          diameter={16}
+          noShadow
+          borderWidth={0}
```

Comment:
I guess, since diameter is small, initials should be smaller.
`initialsSize={10}`
What do you think? :)